addons/mail/static/src/components/file_uploader/file_uploader.js

```diff
@@ -101,7 +101,10 @@ class FileUploader extends Component {
                 attachment.isTemporary &&
                 attachment.filename === file.name
             );
-
+            if (!uploadingAttachment) {
+                //the current attachment may have been removed while awaiting for the previous fetch.
```

Comment:
If it's a phrase you add uppercase at the start.
If it's not a phrase, you don't need a dot at the end.
```suggestion
                // The current attachment may have been removed while awaiting for the previous fetch.
```